demos/python_demos/multi_camera_multi_person_tracking/tools/run_history_visualize.py

```diff
@@ -0,0 +1,249 @@
+            ht_boxes = []
+            ht_labels = []
+            for obj in active_detections[i]:
+                ht_boxes.append([obj.rect[0], obj.rect[1], obj.rect[2] - obj.rect[0], obj.rect[3] - obj.rect[1]])
+                ht_labels.append(obj.label)
+            distances = mm.distances.iou_matrix(np.array(gt_boxes), np.array(ht_boxes), max_iou=0.5)
+            accs[i].update(gt_labels, ht_labels, distances)
+
+    return accs
+
+
+def match_gt_indices(gt_tracks, history, accs):
+    log.info('Assigning GT IDs to IDs from history...')
+    hist_max_id = find_max_id(history)
+    gt_max_id = find_max_id(gt_tracks)
+    assignment_matrix = np.zeros((gt_max_id + 1, hist_max_id + 1), dtype='int32')
+    for acc in accs:
+        for event in acc.events.values:
+            if event[0] == 'MATCH':
+                gt_id = int(event[1])
+                hist_id = int(event[2])
+                assignment_matrix[gt_id][hist_id] += 1
+    assignment_indices = np.argsort(-assignment_matrix, axis=1)
+    next_missed = -1
+    for i in range(assignment_indices.shape[0]):
+        if assignment_indices[i][0] == 0 and np.amax(assignment_matrix[i]) == 0:
+            assignment_indices[i][0] = next_missed
+            next_missed -= 1
+    for i in range(len(gt_tracks)):
+        used_ids = []
+        for j in range(len(gt_tracks[i])):
+            base_id = gt_tracks[i][j]['id']
+            offset = 0
+            while assignment_indices[gt_tracks[i][j]['id']][offset] in used_ids:
+                offset += 1
+            gt_tracks[i][j]['id'] = assignment_indices[gt_tracks[i][j]['id']][offset]
+            used_ids.append(gt_tracks[i][j]['id'])
+            log.info('Assigned GT ID: {} --> {}'.format(base_id, gt_tracks[i][j]['id']))
+    return gt_tracks
+
+
+def calc_output_video_params(input_sizes, fps, gt, merge_det_gt_windows,
+                             max_window_size=(1920, 1080), stack_frames='vertical'):
+    assert stack_frames in ['vertical', 'horizontal']
+    widths = []
+    heights = []
+    for wh in input_sizes:
+        widths.append(wh[0])
+        heights.append(wh[1])
+    if stack_frames == 'vertical':
+        target_width = min(widths)
+        target_height = sum(heights)
+    elif stack_frames == 'horizontal':
+        target_width = sum(widths)
+        target_height = min(heights)
+    if gt and merge_det_gt_windows:
+        target_width *= 2
+    vis = np.zeros((target_height, target_width, 3), dtype='uint8')
+    target_width, target_height = get_target_size(input_sizes, vis, max_window_size, stack_frames)
+    target_fps = min(fps)
+    return (target_width, target_height), target_fps
+
+
+def main():
+    """Prepares data for the person recognition demo"""
+    parser = argparse.ArgumentParser(description='Multi camera multi person \
+                                                  tracking visualization demo script')
+    parser.add_argument('--videos', type=str, nargs='+',
```

Comment:
It should be `-i` to align with the demo.

Reply:
Fixed.